sparse_strips/vello_common/src/coarse.rs

```diff
@@ -71,46 +87,78 @@ struct Clip {
     pub thread_idx: u8,
 }
 
-/// A bounding box
+/// An axis-aligned bounding box represented by top-left and bottom-right corners.
 ///
-/// The first two values represent the x0 and y0 coordinates, respectively.
-/// The last two values represent the x1 and y1 coordinates, respectively.
-///  x0, y0 — the top-left corner of the bounding box,
-///  x1, y1 — the bottom-right corner of the bounding box.   
-#[derive(Debug, Clone)]
-struct Bbox {
+/// The coordinates are stored as `[x0, y0, x1, y1]` in wide tile coordinates,
+/// where `(x0, y0)` is the top-left corner and `(x1, y1)` is the bottom-right corner.
+#[derive(Debug, Clone, Copy)]
+pub struct Bbox {
```

Comment:
Do you think we should rename this to `WideTilesBbox` or similar to reflect that it must always use wide tile coordinates?

Reply:
Yes, good idea! I was thinking of making it more generic, and I believe I already have that case, which I can add later as a follow-up. But for now, I’m fine with renaming it to keep things clear.

Fixed.